models/endpoints.models.js

```diff
@@ -0,0 +1,8 @@
+const fs = require("fs/promises");
```

Comment:
great job, but we don't need a model for this! this functionality can be in the controller :)